packages/app/src/api.ts

```diff
@@ -132,6 +140,12 @@ export function initializeApp(
     });
   }
 
+  options ||= getDefaultAppConfig();
```

Comment:
First time seeing this syntax. Thank you for the educations 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_OR_assignment

Reply:
That was James, heh.

Reply:
👍 Just mind that it overrides all falsy values—false, 0, "", null, undefined, NaN, etc. This can be surprise folk, especially with ints. 

You can use the nullish form, `??=` if you only want to override null & undefined.